Makefile

```diff
@@ -256,8 +256,12 @@ TEST_BRIDGED_TAP_BIN?=$(BINPATH)/test-bridged-tap
 $(TEST_BRIDGED_TAP_BIN): $(shell find internal/cmd/test-bridged-tap -name *.go) $(GOMOD) $(GOSUM)
 	go build -o $@ $(CURDIR)/internal/cmd/test-bridged-tap
 
+LOOPBACK_BIN?=$(BINPATH)/loopback
+$(LOOPBACK_BIN): 
+	GOBIN=$(dir $@) GO111MODULE=off go get -u github.com/containernetworking/plugins/plugins/main/loopback
```

Comment:
Is it possible to pin the version of loopback? I believe go get is going to fetch the latest version.

Reply:
Yea, go get will get version @ latest . I was following how we currently get/install other container networking plugins: https://github.com/firecracker-microvm/firecracker-containerd/blob/3c257503cc1fa13b3dc941b823a0627e5c77e88f/Makefile#L235-L249

Reply:
Oh okay. I'm fine doing that in a separate PR in this case.